### PR TITLE
Fix the expression of some comments

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
I found some inaccurate comments and fixed them.
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/util/config/joinconfiguration.go [hunk 1]
```diff
@@ -45,7 +45,11 @@ func SetJoinDynamicDefaults(cfg *kubeadmapi.JoinConfiguration) error {
 	return nil
 }
 
-// JoinConfigFileAndDefaultsToInternalConfig
+// JoinConfigFileAndDefaultsToInternalConfig takes a path to a config file and a versioned configuration that can serve as the default config
+// If cfgPath is specified, defaultversionedcfg will always get overridden. Otherwise, the default config (often populated by flags) will be used.
+// Then the external, versioned configuration is defaulted and converted to the internal type.
+// Right thereafter, the configuration is defaulted again with dynamic values (like IP addresses of a machine, etc)
+// Lastly, the internal config is validated and returned.
```
Comment (neolit123) [inline cmd/kubeadm/app/util/config/joinconfiguration.go]: i don't think we need this commend has to be that detailed. :)
once sentence would suffice.

we are also lacking a similar comment in initconfiguration.go

**Review by timothysc — APPROVED** [review]
/approve

**Pull request merged**